### ISSUE TITLE
Website is missing lang attribute

### ISSUE THREAD

**Issue by doug-wade** [doug-wade/tybalt#407]
Chrome lighthouse is reporting

> If a page doesn't specify a lang attribute, a screen reader assumes that the page is in the default language that the user chose when setting up the screen reader. If the page isn't actually in the default language, then the screen reader might not announce the page's text correctly. [Learn more about the lang attribute](https://dequeuniversity.com/rules/axe/4.7/html-has-lang).

<img width="804" alt="This is a screenshot of the issue quoted above" src="https://github.com/doug-wade/tybalt/assets/3477707/19467a5e-e6ca-4957-ae47-eaf917247b02">

**Comment by doug-wade** [comment]
We'll need to add the new attribute right here: https://github.com/doug-wade/tybalt/blob/master/packages/website/index.html#L2

**Comment by doug-wade** [comment]
Verified this is resolved!
